src/videojs-http-streaming.js

```diff
@@ -825,7 +824,8 @@ class VhsHandler extends Component {
 
       setupEmeOptions({
         player: this.player_,
-        sourceKeySystems: this.source_.keySystems,
+        // eslint-disable-next-line
+        sourceKeySystems: this.source_.keySystems || this.source_.key_systems,
```

Comment:
eme appears to clone `key_systems` into `keySystems`, but without the eme plugin that won't happen.

Reply:
I think we can ignore `key_systems` here as contrib-eme only expects `keySystems`.

Reply:
Without eme though we won't pass any options in and it won't seem like a drm source at all. That means that the check I added in `setupEmeOptions` won't happen.